lib/langchain/llm/openai.rb

```diff
@@ -61,13 +61,17 @@ def complete(prompt:, **params)
     # Generate a chat completion for a given prompt
     #
     # @param prompt [String] The prompt to generate a chat completion for
+    # @param messages [Array] The messages that have been sent in the conversation
     # @return [String] The chat completion
     #
-    def chat(prompt:, **params)
-      parameters = compose_parameters DEFAULTS[:chat_completion_model_name], params
+    def chat(prompt: "", messages: [], **params)
+      return if prompt.empty? && messages.empty?
```

Comment:
Should we raise an error? Returning `nil` would be rather confusing to a developer.

Reply:
yeah, an error is better